tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -207,6 +207,76 @@ error: Cannot access private member \\"private_static_method\\" of \\"Foo\\"
 
  
  
+Tests 1 failed (1)
+Test Files 1 failed (1)
+Duration <DURATION>"
+`;
+
+exports[`access_private_apis.test.w 1`] = `
+"error: Class \\"subdir2.MyPrivateClass\\" is private
+  --> ../../../examples/tests/invalid/access_private_apis.test.w:3:5
+  |
+3 | new subdir2.MyPrivateClass();
+  |     ^^^^^^^^^^^^^^^^^^^^^^ Class \\"subdir2.MyPrivateClass\\" is private
+  |
+  --> ../../../examples/tests/invalid/subdir2/file.w:1:7
+  |
+1 | class MyPrivateClass {}
+  |       -------------- defined here
```

Comment:
This is pretty cool IMO - the error message is showing snippets from different files!